app/views/shared/_back_link.erb

```diff
@@ -0,0 +1,3 @@
+<nav aria-label="back link">
```

Comment:
Shame we have to do this, seems really odd!

Reply:
Just done some browsing and found some other GOV sites that have done the same 👍

Reply:
Like this one https://user-auth.apply-to-visit-or-stay-in-the-uk.homeoffice.gov.uk/auth/realms/ExternalChecker/protocol/openid-connect/auth?response_type=code&client_id=external-checker-rtwr&scope=openid%20email%20profile&nonce=9877cf3b67059fe4a00560b78504e114&state=2c5cbaa525fcb76782a361162994c841&redirect_uri=https%3A%2F%2Fright-to-rent.service.gov.uk%2Frtr-view%2Fchecker-details%2Fredirection